lonabot/database.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, filename):
                       'UserID INTEGER PRIMARY KEY,'
                       'Delta INTEGER NOT NULL)')
 
+            c.execute('CREATE TABLE TimeZones('
```

Comment:
I think it would have been a better idea to add a new column to the `TimeDelta` table, since that already exists to store per-user delta information.